pkg/sloop/server/server.go

```diff
@@ -57,7 +57,7 @@ func RealMain() error {
 	factory := &badgerwrap.BadgerFactory{}
 
 	storeRootWithKubeContext := path.Join(conf.StoreRoot, kubeContext)
-	db, err := untyped.OpenStore(factory, storeRootWithKubeContext, time.Duration(1)*time.Hour)
+	db, err := untyped.OpenStore(factory, storeRootWithKubeContext, time.Duration(1)*time.Hour, conf.BadgerMaxTableSize, conf.BadgerKeepL0InMemory, conf.BadgerVLogFileSize, conf.BadgerVLogMaxEntries, conf.BadgerUseLSMOnlyOptions)
```

Comment:
thoughts on using `storemanager.Config` to pass these?

Reply:
good call, added